espnet/nets/pytorch_backend/conformer/attention.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright 2020 Johns Hopkins University (Shinji Watanabe)
+#                Northwestern Polytechnical University (Pengcheng Guo)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+"""Multi-Head Attention layer definition."""
+
+import math
+
+import numpy
+import torch
+from torch import nn
+
+
+class MultiHeadedAttention(nn.Module):
```

Comment:
This class looks same to https://github.com/espnet/espnet/blob/cc75da63b4fb3975e01b6ea3e3ba7580a0c99384/espnet/nets/pytorch_backend/transformer/attention.py#L16

Can you delete this class and reuse that class?

Reply:
Sure, I am moving convolutional block as an option to transformer's encoder now. I will merge conformer related codes into transformer. Thanks so much for your suggestions.